config/core/configmaps/network.yaml

```diff
@@ -37,42 +37,12 @@ data:
     # this example block and unindented to be in the data block
     # to actually change the configuration.
 
-    # istio.sidecar.includeOutboundIPRanges specifies the IP ranges that Istio sidecar
-    # will intercept.
+    # OBSOLETE:
```

Comment:
nit: we have historically preferred the term `DEPRECATED` instead of `OBSOLETE`

Reply:
Done.